app/controllers/appointments_controller.rb

```diff
@@ -34,7 +34,10 @@ def work_order_reference
   end
 
   def available_appointments
-    AppointmentFetcher.new.call(work_order_reference: work_order_reference)
+    AppointmentFetcher
+      .new
+      .call(work_order_reference: work_order_reference)
+      .map { |appointment| AppointmentPresenter.new(appointment) }
```

Comment:
The previous commit mapped these inside `AppointmentFetcher` and this one puts it back in the controller. Is it worth fixing that up?

Reply:
:+1: